code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -801,12 +801,12 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	shell_speed = 2
 	accuracy = 40
 	accurate_range = 20
-	max_range = 30
+	max_range = 40
 	damage = 50
 	penetration = 80
 
 /datum/ammo/rocket/drop_nade(turf/T)
-	explosion(T, -1, 3, 5, 5)
+	explosion(T, 1, 3, 5, 5)
```

Comment:
this is also 3x3 i think, you would need 0 for it to be only on the hitting tile

Reply:
I'll look into it